javascript/implementation/KeySet.ts

```diff
@@ -69,12 +69,12 @@ export class KeySet extends Container {
      * @param asOf
      * @returns an async iterator across everything in the key set, with values returned as pairs of Key, Key
      */
-    entries(asOf?: AsOf): AsyncGenerator<[UserKey, UserKey], void, unknown> {
+    entries(asOf?: AsOf): AsyncGenerator<[ScalarKey, ScalarKey], void, unknown> {
         const thisSet = this;
         return (async function* () {
             const entries = await thisSet.database.store.getKeyedEntries(thisSet.address, asOf);
             for (const [key, entry] of entries) {
-                const efffectiveKey = <UserKey>entry.effectiveKey;
+                const efffectiveKey = <ScalarKey>entry.storageKey;
```

Comment:
one too many fs for the variable name :P also may want to change the variable name anyway?

Reply:
So, I'm impressed that you caught that.